packages/shorebird_cli/test/src/extensions/string_test.dart

```diff
@@ -1,16 +1,61 @@
+import 'package:mason_logger/mason_logger.dart';
 import 'package:shorebird_cli/src/extensions/string.dart';
 import 'package:test/test.dart';
 
 void main() {
-  test('IsNullOrEmpty', () async {
-    expect(null.isNullOrEmpty, isTrue);
-    expect(''.isNullOrEmpty, isTrue);
-    expect('test'.isNullOrEmpty, isFalse);
+  group('IsNullOrEmpty', () {
```

Comment:
```suggestion
  group('isNullOrEmpty', () {
```

Reply:
Corrected this to `NullOrEmpty` to reflect the extension name.